.circleci/config.yml

```diff
@@ -12,7 +12,7 @@ jobs:
           path: result.xml
   MacOS:
     macos:
-      xcode: "10.2.1"
+      xcode: "11.0"
```

Comment:
Why we don't go to Xocde 11.1 it's available in CircleCI? https://circleci.com/docs/2.0/testing-ios/

Reply:
It broke with 11.1. You have to choose 11.1.0. Then it works :)

Reply:
Awesome!